var/spack/repos/builtin/packages/py-pygments/package.py

```diff
@@ -29,10 +30,13 @@ class PyPygments(PythonPackage):
     version("2.0.1", sha256="5e039e1d40d232981ed58914b6d1ac2e453a7e83ddea22ef9f3eeadd01de45cb")
     version("2.0.2", sha256="7320919084e6dac8f4540638a46447a3bd730fca172afc17d2c03eed22cf4f51")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
```

Comment:
Auto-detection was incorrect here, this is a pure-Python package. Auto-detection was looking at testing code for example files from every language.